examples/basic-typescript/src/app/[[...path]]/page.tsx

```diff
@@ -3,21 +3,20 @@ import { notFound } from 'next/navigation'
 import { Page as MakeswiftPage } from '@makeswift/runtime/next'
 import { getSiteVersion } from '@makeswift/runtime/next/server'
 
-import { client } from '@/lib/makeswift/client'
+import { client } from '@/makeswift/client'
```

Comment:
@jamesqquick Can we keep the directory structure of this example and update the docs instead? I agree 100% that these two should be synchronized. `@/lib/makeswift` is not only our standard convention in the rest of the examples, though (all of which can be used to bootstrap your project through the CLI), but also our convention in Catalyst. It also makes it easier to distinguish between a global (`@makeswift/runtime/next`) and a local import (`@/lib/makeswift/client`).

Reply:
Alright, updated to go back to the original folder structure. We'll worry about updating docs later on